options.c

```diff
@@ -751,7 +751,7 @@ static void toggle_repeat(void *data)
 }
 
 static const char * const replaygain_names[] = {
-	"disabled", "track", "album", "track-preferred", "album-preferred", NULL
+	"disabled", "track", "album", "track-preferred", "album-preferred", "smart", NULL
 };
 
```

Comment:
Need to increment the "4" in `set_replaygain()` and the "5" in `toggle_replaygain()`

Reply:
good catch. IMO it would be better do something like:

<pre lang="C">static const size_t replaygain_names_len = sizeof(replaygain_names) / sizeof(replaygain_names[0]) - 1;</pre>

and use that instead of hardcoding the length in the other functions.